simplq/src/api/requestFactory/index.js

```diff
@@ -17,3 +17,4 @@ import * as QueueRequestFactory from './queue';
 import * as TokenRequestFactory from './token';
 
 export { QueueRequestFactory, TokenRequestFactory };
+export { getQueueStatus } from './queue';
```

Comment:
I recognized the idea, just wanted to make it shorter and more uniform.

```js
import * as RequestFactory from 'api/requestFactory';

RequestFactory.getQueueStatus(queueId)
RequestFactory.getToken(tokenId)

// instead of 
QueueRequestFactory.getQueueStatus(queueId)
TokenRequestFactory.getToken(tokenId)
```